xml/decoders/Doehler_Haass_fw3.12_DH05-10C_12-16-18-21-22A_PD05-12A.xml

```diff
@@ -501,26 +518,42 @@
           <label xml:lang="de">Rangier-|gang</label>
           <label xml:lang="ca">Marxa de maniobres</label>
         </output>
-        <size length="24.2" width="11.0" height="2.4" units="mm"/>
+        <size length="11.7" width="8.5" height="1.8" units="mm"/>
         <protocols>
           <protocol>dcc</protocol>
         </protocols>
       </model>
-      <model model="DH06A (firmware 3.12.050+)" lowVersionID="50" highVersionID="50" numOuts="3" numFns="16" productID="60" comment="DH06A for Köf with no analog support, with update from May 2020" maxInputVolts="30V" maxMotorCurrent="0.3A" maxTotalCurrent="0.3A">
-        <output name="1" label="Front|Light" maxcurrent="20mA"/>
-        <output name="2" label="Rear|Light" maxcurrent="20mA"/>
-        <output name="3" label="Onboard|LED" maxcurrent="20mA"/>
-        <output name="4" label="|"/>    <!-- not available for this decoder -->
-        <output name="5" label="|"/>    <!-- not available for this decoder -->
-        <output name="6" label="|"/>    <!-- not available for this decoder -->
-        <output name="7" label="Dimmed|Lights"/>
-        <output name="8" label="Shunting|Speed"/>
-        <size length="16" width="9.3" units="mm"/>
+      <model model="PD12A (firmware 3.12.050+)" lowVersionID="50" highVersionID="50" numOuts="4" numFns="16" productID="130" comment="PD12A-0 / PD12A-2 / PD12A-3 / PD12A-4 with update from May 2020" maxInputVolts="30V" maxMotorCurrent="1A" maxTotalCurrent="1A" connector="PluX12">
+        <output name="1" label="Front|Light" maxcurrent="150mA">
+          <label xml:lang="de">LV</label>
```

Comment:
Is `LV` a correct German translation? I'm not against it, but if you are unsure, it might be better to leave it untranslated and let someone else later translate it to German. I don't speek German myself so I don't know.

Reply:
Yes it is, straight from the manufacturer's manual:
LV, LR Licht vorwärts, rückwärts
It is a very usual abbreviation and all around the place in  JMRI's decoder definition. BTW, was already in the file as is.
No change.